### ISSUE TITLE
Suggestion: Use UDPipe for support for 40 languages

### ISSUE THREAD

**Issue by EmilStenstrom** [clips/pattern#154]
**Context:** I really like the look of pattern, but I'm looking for something that supports *swedish*. But since this isn't one of the big languages I doubt it will be added any time soon, looking at how much manual job there seems to be adding a new language to pattern. 

A couple of days ago I found out about [udpipe](https://github.com/ufal/udpipe), I library that does tokenization, tagging, lemmatization and dependency parsing of 40 languages, with one API. It builds on open data, has a permissive license, and anyone can download free pre-trained models.

**Idea:** What would happen if you used udpipe to add the 35 languages you don't yet support? I understand this is a big project, but looking at open issues I see that a lot of the open issues are related to language specific things; things that you could get for free if you depended in a bigger library.

So basically, my suggestion is: integrate udpipe to pattern as a way to support more languages.
